### PR TITLE
feature/mention_badge_as_referral_reward

### PR DESCRIPTION
**Pull request by vyvil**
This PR adjust texts on referral rewards based on this case description:
**I got invited, accepted the invitation, got rewarded with 250 PLR and then booom — new badge** ([ref](https://pillarwallet.slack.com/archives/CH4HBT15Y/p1590743753141900?thread_ts=1590488827.139300&cid=CH4HBT15Y)).

### CODE REVIEW

File: src/screens/Users/VerifiedModal.js [hunk 1]
```diff
@@ -59,11 +59,10 @@ const VerifiedModal = (props: Props) => {
...
 
-  const text = (isPillarRewardCampaignActive && rewardText) ?
-    `You are now able to invite friends via ${referralMethod}. You will receive ${rewardText} and a badge 
-     for each friend installed the app with your referral link.` :
-    `You are now able to invite friends via ${referralMethod}. Start receiving badges for friends 
-     installed Pillar with your referral link!`;
+  const text = (isPillarRewardCampaignActive && rewardText)
+    ? `You are now able to invite friends via ${referralMethod}. You will receive ${rewardText} for each friend` +
+    'installed the app with your referral link.'
+    : `You are now able to invite friends via ${referralMethod}.`;
```
Comment (graszka22) [inline src/screens/Users/VerifiedModal.js:65]: If there is no ongoing campaign we'll show `You are now able to invite friends via ${referralMethod}.` and we won't mention badges. But we award badges even if there is no campaign, correct? So why are we deleting this: `Start receiving badges for friends ...`

File: src/screens/ReferFriends/ReferralIncomingReward.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ class ReferralIncomingReward extends React.PureComponent<Props> {
     const relatedCampaign = rewardsByCampaign['1world'] ? '1world' : 'pillar';
 
     const rewardText = getRewardText(rewardsByCampaign, relatedCampaign);
-    const isGettingReward = rewardText;
+    const isGettingReward = !!rewardText;
```
Comment (alazarevski) [inline src/screens/ReferFriends/ReferralIncomingReward.js:88]: {!!isGettingReward && <RewardBadge source={rewardBadge} />}
this is already cast at line 101

**Comment by IAmKio** [comment]
Hi @vyvil jus wondering what the status of this PR is?

File: src/screens/Users/VerifiedModal.js (hunk 1 above)
Comment (vyvil) [reply src/screens/Users/VerifiedModal.js:65]: True, will revert it back :)

File: src/screens/ReferFriends/ReferralIncomingReward.js (hunk 2 above)
Comment (vyvil) [reply src/screens/ReferFriends/ReferralIncomingReward.js:88]: I'll remove '!!' in later lines. Keeping it as boolean here :)

**Comment by vyvil** [comment]
> Hi @vyvil jus wondering what the status of this PR is?

Just updated it according to reviews. It's ready to be checked again :)

**Pull request merged**